enterprise/server/auth/auth.go

```diff
@@ -223,9 +224,10 @@ type oidcAuthenticator struct {
 	slug         string
 }
 
-func extractToken(issuer string, idToken *oidc.IDToken) (*userToken, error) {
+func extractToken(issuer string, slug string, idToken *oidc.IDToken) (*userToken, error) {
```

Comment:
nit: you can collapse this to:

func extractToken(issuer, slug string, idToken *oidc.IDToken) (*userToken, error) {

Reply:
Done